From 2db90b14131b52f17615d812c3ac3bae6a150a09 Mon Sep 17 00:00:00 2001
From: Ky-Anh Huynh <kyanh@theslinux.org>
Date: Fri, 12 Apr 2013 08:05:41 +0000
Subject: [PATCH] Add new module :xvni

This module is a big table of predefined words for VNI users.
It is genereated from https://github.com/icy/myquartz-scim2uim/ \
  blob/master/src/VNI.txt.in
---
 uim-vi/src/uim-1.8.5/pixmaps/Makefile.am | 4 ++--
 uim-vi/src/uim-1.8.5/scm/Makefile.am     | 3 ++-
 2 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/uim-vi/src/uim-1.8.5/pixmaps/Makefile.am b/uim-vi/src/uim-1.8.5/pixmaps/Makefile.am
index 46a4324..c3a2792 100644
--- a/uim-vi/src/uim-1.8.5/pixmaps/Makefile.am
+++ b/uim-vi/src/uim-1.8.5/pixmaps/Makefile.am
@@ -25,12 +25,12 @@ SVGS += direct_input_dark_background.svg \
 SVGS += im_switcher.svg uim-dict.svg
 
 # input method substitutions
-IM_SUBST_SVGS = xtelex.svg viqr.svg ipa-x-sampa.svg scim.svg
+IM_SUBST_SVGS = xvni.svg xtelex.svg viqr.svg ipa-x-sampa.svg scim.svg
 
 GENERATED_SVGS = $(IM_SUBST_SVGS)
 SVGS += im_subst.svg $(GENERATED_SVGS)
 
-IM_SUBST_PNGS = xtelex.png viqr.png ipa-x-sampa.png scim.png
+IM_SUBST_PNGS = xvni.svg xtelex.png viqr.png ipa-x-sampa.png scim.png
 
 SVG_PNGS = unknown.png \
            direct.png \
diff --git a/uim-vi/src/uim-1.8.5/scm/Makefile.am b/uim-vi/src/uim-1.8.5/scm/Makefile.am
index 636691f..2256e1d 100644
--- a/uim-vi/src/uim-1.8.5/scm/Makefile.am
+++ b/uim-vi/src/uim-1.8.5/scm/Makefile.am
@@ -19,6 +19,7 @@ SCM_FILES = plugin.scm im.scm im-custom.scm lazy-load.scm init.scm \
  generic-predict.scm predict-custom.scm \
  viqr.scm \
  xtelex.scm \
+ xvni.scm \
  ipa-x-sampa.scm \
  m17nlib.scm m17nlib-custom.scm \
  uim-module-manager.scm \
@@ -45,7 +46,7 @@ EXTRA_DIST = $(OTHER_SCM_FILES)
 
 ETAGS_ARGS=$(SCM_FILES) $(GENERATED_SCM_FILES)
 
-module_names = "m17nlib" "viqr" "xtelex" "ipa-x-sampa"
+module_names = "m17nlib" "viqr" "xvni" "xtelex" "ipa-x-sampa"
 
 UIM_MODULE_MANAGER = $(top_builddir)/uim/uim-module-manager
 UIM_MODULE_MANAGER_ENV = \
-- 
1.8.3

